plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java

```diff
@@ -521,4 +529,51 @@ public static String getStateFragmentName(StateFragment capellaStateFragment) {
     }
     return null;
   }
+  
+
+  public static boolean isOpenedRepresentation() {
+    IWorkbenchPage activePage = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage();
+    if (activePage == null) {
+      return false;
+    }
+    for (IEditorReference ref : activePage.getEditorReferences()) {
```

Comment:
Can this function make use of the following one (getOpenedRepresentation) to have less duplicated code?

Reply:
i'll check